microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -35,24 +67,43 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
 <!-- ====================================================================== -->
 ## Before you begin
 
-To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials. Use the following steps to create API credentials.
+To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials.  Use the following steps to create API credentials.
 
 1. Visit Microsoft Partner Center and sign in to the account that you used to publish an add-on.
 
 1. Under the **Microsoft Edge** program, select **Publish API**.
 
 1. On the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes to finish.
 
-   The API credentials have now been created; you've enabled or renewed the API.  The **Client ID**, **Client secret**, **Expiry date**, and **Access token URL** are now displayed on the Publish APIs page:
+   The API credentials have now been created; you've enabled or renewed the API.  The following items are now displayed on the **Publish API** page:
+   * **Client ID**
+   * **Client secret** (for REST Publish API v1) or **API key** (for REST Publish API v2)
+   * **Expiry date**
+   * **Access token URL** (for REST Publish API v1)
+
+   ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)<!-- todo: update per v2; maybe show a v2 png and then a v1 png -->
 
-   ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)
+1. Write down the following:
+   * The **Client ID**.
+   * The **Client secret** (REST Publish API v1) or **API key** (REST Publish API v2).
+   * The **Access token URL** (REST Publish API v1).
 
-1. Write down the **Client ID**, **Client secret** and the **Access token URL**.  You'll use these values in the next step, to get an access token.
+   You'll use these values in the next step, to get an access token.
+
+<!-- todo: update steps to mention selecting v2 vs v1.
```

Comment:
is there a UI control in Partner Center page to select v1 vs v2 API?   

IMO, as a user/reader, this article needs a Partner Center screenshot, to make article concrete instead of abstract.

Can you paste rough screenshot here or in chat?  to understand this article, i need to see Partner Center - i can then probably describe that UI in text, w/o necessarily including png in article.  If you provide clean png, i might include in article.

To provide publishable screenshot: 
* best is whole-window, not fullscreen, not just portion of window.
* Make window as narrow as feasible while still looking normal.  
* I'll delete any personal info not needed.
* Screenshots get outdated, but that's understood throughout industry dev docs.  Avoid if highly volatile per a soon, major redesign.

@bhuvanapriyap @yvsubhash

* What's the title of the page that contains the UI control to select v1 vs v2?  
* What's the label of the UI control?  
* Is the UI control an Option button, or a Button?

Reply:
@mikehoffms , we need to change v2 as v1.1 in the document

Reply:
globally in article, changed v2 to v1.1

Reply:
closing Outdated comments.  added todo notes in .md; will create fresh Comment at each todo note